Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM alpine AS builder
 RUN mkdir /usr/local/src && apk add binutils --no-cache\
-        build-base \
+        linux-headers \
+	build-base \
```

Comment:
Inconsistent indentation...

Reply:
I'm going to move those definitions to main repo.

who knows, maybe I'll add some check ))